src/Sulu/Component/Webspace/Analyzer/Attributes/PortalInformationRequestProcessor.php

```diff
@@ -52,7 +53,7 @@ public function process(Request $request, RequestAttributes $requestAttributes)
         if ($attributes['localization']) {
             $localization = $attributes['localization'];
             $attributes['locale'] = $localization->getLocale();
-            $request->setLocale($localization->getLocale());
+            $request->setLocale($localization->getLocale(Localization::LCID));
```

Comment:
Sadly we can not change this as it would be a bc break for all accessing it currently and doing checks on it.

Reply:
If I understand you correctly I think at current state you need to use `messages.de_de.yaml` instead of `messages.de_DE.yaml` for translations. Because I think translations are currently the only services having a fallback locale?

Reply:
Correct.